src/megameklab/com/printing/PrintRecordSheet.java

```diff
@@ -268,14 +269,15 @@ public int print(Graphics graphics, PageFormat pageFormat, int pageIndex) throws
             node.paint(g2d);
             /* Testing code that outputs the generated svg
             try {
-                Transformer transformer = TransformerFactory.newInstance().newTransformer();
-                Result output = new StreamResult(new File("out.svg"));
-                Source input = new DOMSource(svgDocument);
+                javax.xml.transform.Transformer transformer = javax.xml.transform.TransformerFactory.newInstance().newTransformer();
+                javax.xml.transform.Result output = new javax.xml.transform.stream.StreamResult(new File("out.svg"));
+                javax.xml.transform.Source input = new javax.xml.transform.dom.DOMSource(svgDocument);
```

Comment:
I changed this so I wouldn't have to keep messing with the imports when I enable this to troubleshoot the svg output.